packages/bsky/src/api/app/bsky/graph/getList.ts

```diff
@@ -68,7 +70,29 @@ const hydration = async (
       params.hydrateCtx,
     ),
   ])
-  return mergeStates(listState, profileState)
+  const bidirectionalBlocks = await maybeGetBlocksForReferenceList({
+    ctx,
+    params,
+    skeleton,
+    listState,
+  })
+  return [listState, profileState, { bidirectionalBlocks }].reduce(
+    mergeStates,
+    {} as HydrationState,
+  )
```

Comment:
The `mergeManyStates()` helper will reduce for you!  May need to export it, but it lives right next to `mergeStates()`.